hamocc/hamocc4bcm.F90

```diff
@@ -124,13 +124,15 @@ SUBROUTINE HAMOCC4BCM(kpie,kpje,kpke,kbnd,kplyear,kplmon,kplday,kldtday,&
 
 
 !--------------------------------------------------------------------
-! Increment bgc time step counter of run (initialized in INI_BGC).
+! Increment bgc time step counter of run (initialized in HAMOCC_INIT).
 !
       ldtrunbgc = ldtrunbgc + 1
 
 
 !--------------------------------------------------------------------
 ! Increment bgc time step counter of experiment (initialized if IAUFR=0).
+! 2021-06-18 : IAUFR does not seem to be defined. ldtbgc is defined in
+!              mo_control_bgc and initialized in hamocc_init.
```

Comment:
Maybe we should rather remove the text "(initialized if IAUFR=0)" instead of adding a comment that the comment seems to be wrong?

Reply:
Agree. I just wanted to confirm that was actually incorrect. I will remove the text.